src/main/utils/PlaybackAPI.js

```diff
@@ -134,6 +138,17 @@ class PlaybackAPI {
     this._save();
   }
 
+  _setPauseAfter(mode) {
+    if (mode) {
+      Emitter.sendToGooglePlayMusic('toast:pauseAfter', null);
```

Comment:
can we change the scope here from `toast:` to `pauseAfter:show` and `pauseAfter:hide`? Seems like it might make more sense in the long run.